src/c++/perf_analyzer/README.md

```diff
@@ -1,752 +1,167 @@
-    Avg HTTP time: 48166 usec (send/recv 264 usec + response wait 47902 usec)
-  Server:
-    Inference count: 498
-    Execution count: 498
-    Successful request count: 498
-    Avg request latency: 45602 usec (overhead 39 usec + queue 33577 usec + compute input 217 usec + compute infer 11753 usec + compute output 16 usec)
-
-Inferences/Second vs. Client Average Batch Latency
-Concurrency: 1, throughput: 67.8 infer/sec, latency 14710 usec
-Concurrency: 2, throughput: 89.8 infer/sec, latency 22280 usec
-Concurrency: 3, throughput: 80.4 infer/sec, latency 37283 usec
-Concurrency: 4, throughput: 83 infer/sec, latency 48064 usec
-```
-
-## Understanding The Output
-
-### How Throughput is Calculated
-
-Perf Analyzer calculates throughput to be the total number of requests completed
-during a measurement, divided by the duration of the measurement, in seconds.
-
-### How Latency is Calculated
-
-For each request concurrency level perf_analyzer reports latency and
-throughput as seen from the *client* (that is, as seen by
-perf_analyzer) and also the average request latency on the server.
-
-The server latency measures the total time from when the request is
-received at the server until the response is sent from the
-server. Because of the HTTP and GRPC libraries used to implement the
-server endpoints, total server latency is typically more accurate for
-HTTP requests as it measures time from first byte received until last
-byte sent. For both HTTP and GRPC the total server latency is
-broken-down into the following components:
-
-- *queue*: The average time spent in the inference schedule queue by a
-  request waiting for an instance of the model to become available.
-- *compute*: The average time spent performing the actual inference,
-  including any time needed to copy data to/from the GPU.
-
-The client latency time is broken-down further for HTTP and GRPC as
-follows:
+Triton Performance Analyzer is CLI tool which can help you optimize the
+inference performance of models running on Triton Inference Server by measuring
+changes in performance as you experiment with different optimization strategies.
 
-- HTTP: *send/recv* indicates the time on the client spent sending the
-  request and receiving the response. *response wait* indicates time
-  waiting for the response from the server.
-- GRPC: *(un)marshal request/response* indicates the time spent
-  marshalling the request data into the GRPC protobuf and
-  unmarshalling the response data from the GRPC protobuf. *response
-  wait* indicates time writing the GRPC request to the network,
-  waiting for the response, and reading the GRPC response from the
-  network.
+<br>
 
-Use the verbose (-v) option to perf_analyzer to see more output,
-including the stabilization passes run for each request concurrency
-level.
+# Features
 
-## Measurement Modes
+### Inference Load Modes
 
-### Time Windows
+- [Concurrency Mode](docs/request_sending_modes.md#concurrency-mode) simlulates
```

Comment:
request_sending_modes.md does not exist?

Reply:
Fixed 👍